app/controllers/vacancies_controller.rb

```diff
@@ -49,14 +49,14 @@ def set_headers
 
   def trigger_search_performed_event
     fail_safe do
-      vacancy_ids = @vacancies_search.vacancies.map(&:id)
+      vacancy_ids = @vacancies_search.vacancies.unscope(:order).pluck(:id)
```

Comment:
The idea of sending ALL the results to analytics defeats the pagination point.
The memory impact of mapping through all the unpaginated results here is massive for the request lifespan.

We use `pluck` to mitigate it, so we don't need to instantiate all the search result objects but directly get a list of IDs from DB.

This `pluck` call causes issues with the order criteria, but as we don't need ordered results here, we can remove the ordering and achieve a way more efficient way to get the result IDs.